src/firebolt_db/firebolt_dialect.py

```diff
@@ -112,6 +113,10 @@ def create_connect_args(self, url: URL) -> Tuple[List, Dict]:
         parameters = dict(url.query)
         if "account_name" in parameters:
             kwargs["account_name"] = parameters.pop("account_name")
+        if "use_token_cache" in parameters:
+            kwargs["use_token_cache"] = bool(
```

Comment:
Please add a comment about this conversion.
It's not obvious what comes in parameters and how it's transformed